lib/Zonemaster/Backend/DB/MySQL.pm

```diff
@@ -41,6 +42,11 @@ sub dbh {
         my $password = $self->config->MYSQL_password();
 
         $log->notice( "Connecting to MySQL: database=$database host=$host user=$user" ) if $log->is_notice;
+
+        if ( untaint_ipv6_address( $host ) ) {
+            $host = "[$host]";
+        }
```

Comment:
Would it make sense to swap the `log` and the `if` statements so the value used for the `host` is caught in the log ?

Reply:
To me it makes more sense to report the plain format rather than how it's represented in the DBD::mysql DSN. That way IPv6 addresses are reported the same way when using MariadDB and PostgreSQL.

Another option would be to report the DSN and username in the same format for all database adapters.

Reply:
Ok that makes sense indeed.